open-metadata-implementation/access-services/digital-service/digital-service-api/src/main/java/org/odpi/openmetadata/accessservices/digitalservice/rest/DigitalServiceRequestBody.java

```diff
@@ -0,0 +1,74 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.digitalservice.rest;
+
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.odpi.openmetadata.accessservices.digitalservice.properties.DigitalService;
+
+import java.util.Objects;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.PUBLIC_ONLY;
+
+/**
+ * The request body of creating the new digital service
+ *
+ */
+
+public class DigitalServiceRequestBody extends DigitalServiceOMASAPIRequestBody {
```

Comment:
Do we need a request body - or could the digital serivce be passed as the rest API body

Reply:
I had a foot in both camps, I'll address this with the first drop  of the client.